.gitignore

```diff
@@ -65,3 +65,4 @@ target/
 pyuvdata/data/
 pyuvdata/data/test/
 pyuvdata/GIT_INFO
+uvdata/data/test/*
```

Comment:
Why is this needed? Do tests need to write to a different folder than pyuvdata/data/test/ ?